play.lic

```diff
@@ -0,0 +1,35 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#play
+=end
+class Play
+  def initialize
+    @no_play_scripts = get_settings.sanowret_no_use_scripts
```

Comment:
❓ Is the list of scripts always going to be the same as for sanowret or is that just a quick hack for now?

Reply:
Its a quick hack. There should be a play_no_use_scripts